Portal/pom.xml

```diff
@@ -37,30 +37,30 @@ knowledge of the CeCILL-B license and that you accept its terms.
     <groupId>fr.insalyon.creatis</groupId>
     <artifactId>vip</artifactId>
     <packaging>pom</packaging>
-    <version>1.14</version>
+    <version>1.15</version>
     <name>VIP</name>
     
     <properties>
-        <vip.core.version>1.14</vip.core.version>
-        <vip.portal.version>1.14</vip.portal.version>
-        <vip.docs.version>1.6</vip.docs.version>
-        <vip.application.version>1.12</vip.application.version>
-        <vip.datamanager.version>1.12</vip.datamanager.version>
-        <vip.datamanager-applet.version>1.13</vip.datamanager-applet.version>
-        <vip.gatelab.version>1.9</vip.gatelab.version>
-        <vip.gatelab-applet.version>1.13</vip.gatelab-applet.version>
+        <vip.core.version>1.15</vip.core.version>
+        <vip.portal.version>1.15</vip.portal.version>
+        <vip.docs.version>1.15</vip.docs.version>
```

Comment:
Actually, I changed the version of all modules that have been re-compiled with Java 8 because their pom files have changed (as well as the corresponding jars, of course). If you believe these changes should not impact version numbers, I'll set them back to their previous version.